lib/meilisearch/client.rb

```diff
@@ -41,7 +39,10 @@ def delete_index(index_uid)
     def index(index_uid)
       index_object(index_uid)
     end
-    alias get_index index
+
+    def fetch_index(index_uid)
+      index_object(index_uid).fetch_info
```

Comment:
Any reason to keep fetch_info?

Also, no ruby expert, but can't you achieve this with an alias?

Reply:
`fetch_info` is the method in the `Index` class -> `client.index('uid').fetch_info` -> equivalent method in python SDK: https://github.com/meilisearch/meilisearch-python/blob/73bd05ab77b3e4ae48969ccbade8fe874e0e6588/meilisearch/index.py#L69-L79
`fetch_index` is the method in the `Client` class -> `client.fetch_index('uid')` -> equivalent method in python SDK: https://github.com/meilisearch/meilisearch-python/blob/73bd05ab77b3e4ae48969ccbade8fe874e0e6588/meilisearch/client.py#L64-L82

Reply:
Thanks for the clear explanation :)